nulib/meadow
# Replace react-media-player with clover-iiif for viewer dependency

# Summary 
We moved the player formerly known as RMP to https://github.com/samvera-labs/clover-iiif, and updated the NPM package to https://www.npmjs.com/package/@samvera/clover-iiif

# Specific Changes in this PR
- Swapped out the dependency location / import and component name.

# Version bump required by the PR


# Steps to Test
If you can run Meadow successfully, navigate to an audio or video Work and see if the player loads successfully.


# Tested/Verified
